projects/plugins/automattic-for-agencies-client/src/js/components/connection-card/index.jsx

```diff
@@ -1,61 +1,333 @@
+	return (
+		<>
+			<div>
+				<button
+					className="components-button components-button--back is-link"
+					onClick={ onCloseSharingDetailsClick }
+				>
+					<LeftArrow />
+					{ __( 'Go back to site connection', 'automattic-for-agencies-client' ) }
+				</button>
+			</div>
+			<div className={ styles.card__prose }>
+				<h2>What data is synced from your site</h2>
+				<p>
+					Your privacy matters to us. We sync only the necessary data required to provide our
+					Automattic for Agencies portal experiences.
+				</p>
+			</div>
+			<ul className={ styles.checklist }>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Options:</strong> to sync site options data.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Callables:</strong> to sync for callables.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Constants:</strong> to sync for constants.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Full_Sync_Immediately:</strong> to do a full resync of the database.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Updates:</strong> to sync data about plugin, theme, and core updates.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Plugins:</strong> to sync plugin data such as deletions and installations.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Users:</strong> to sync changes to users.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Meta:</strong> to sync meta information users and other relevant objects.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+				<li>
+					<CheckIcon />
+					{ createInterpolateElement(
+						__(
+							'<strong>Stats:</strong> to sync heartbeat stats.',
+							'automattic-for-agencies-client'
+						),
+						{ strong: <strong /> }
+					) }
+				</li>
+			</ul>
+			<div className={ styles.card__prose }>
+				<h2>Data we will not sync</h2>
```

Comment:
Same here, it would need to be translatable.

Reply:
Fixed via https://github.com/Automattic/jetpack/pull/36729/commits/4802b612a786c9d82b46256d18533aaf9210b710